WebApiProxy.Server/MetadataProvider.cs

```diff
@@ -225,7 +225,14 @@ private void AddModelDefinition(Type classToDef)
                 var properties = classToDef.IsGenericType
                                      ? classToDef.GetGenericTypeDefinition().GetProperties()
                                      : classToDef.GetProperties();
-            
+
+		// remove if properties or their types has got ExcludeProxy Attribute (or dummy name)
+		properties = properties.Where(x =>
+                !x.CustomAttributes.Any(z => z.AttributeType.Name.Contains("ExcludeProxy")) && // Property
+                !x.PropertyType.CustomAttributes.Any(y => y.AttributeType.Name.Contains("ExcludeProxy")) // Type

```

Comment:
Avoid magic strings where possible. 

An idea is to either compare to `nameof(ExcludeProxy)` or just compare where types contain `typeof(ExcludeProxy)`